src/configure/helper/mustacheHelper.ts

```diff
@@ -95,6 +95,25 @@ export class MustacheHelper {
                     }
 
                     return "";
+                };
+            },
+            "equals": function () {
+                return function (text: string, render: any) {
```

Comment:
As we discussed, I have already implemented equals in my rightClickScenario PR

Reply:
Needed to evaluate some dynamic validation conditions, removed it from my PR.